geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/dunit/CqPerfDUnitTest.java

```diff
@@ -119,14 +121,11 @@ public void run2() throws CacheException {
         // Create and Execute CQ.
         try {
           CqQuery cq1 = cqService.newCq(cqName, cqDUnitTest.cqs[0], cqa);
-          assertTrue("newCq() state mismatch", cq1.getState().isStopped());
+          assertThat(cq1.getState().isStopped()).withFailMessage("newCq() state mismatch").isTrue();
           cq1.execute();
         } catch (Exception ex) {
-          LogWriterUtils.getLogWriter().info("CqService is :" + cqService);
-          ex.printStackTrace();
-          AssertionError err = new AssertionError("Failed to create CQ " + cqName + " . ");
-          err.initCause(ex);
-          throw err;
+          log.info("CqService is :" + cqService, ex);
```

Comment:
Thanks for cleaning up this test a bit, but if your are cleaning up these lines there is not reason for any test to catch an exception and call Assert.fail. It should just not catch the exception and let the test fail if there is an exception.

Reply:
Sometimes it's helpful (depending on the size of the test) to catch and wrap an exception in order to describe the context in which the exception was thrown—what the test was trying to do at the time. That can make life a little easier for whoever has to diagnose a failure.There are potentially better ways to give context, such as extracting the try/catch block into an expressively-named method, so that the method name in the stack trace describes the context. If I were writing these classes from scratch, I'd use the nicely-named method technique. But with existing tests… I dunno. I don't have a clear sense of how far to impose my preferences onto existing tests.

Reply:
done